applications/nrf5340_audio/src/bluetooth/bt_management/controller_config/bt_mgmt_ctlr_cfg.c

```diff
@@ -12,19 +12,24 @@
 #include <zephyr/task_wdt/task_wdt.h>
 
 #include "macros_common.h"
+#if (CONFIG_BT_LL_ACS_NRF53)
```

Comment:
No need to not include this, we should guard inside this file probably, and not outside it

Reply:
This file is not included when building for other controllers